### ISSUE TITLE
\step command broken with update of `lettrine` package to v2.01

### ISSUE THREAD

**Issue by samcarter8** [SvenHarder/xcookybooky#13]
In its current definition the `\step` macro no longer works with v2.01 of the `lettrine` package. 

This slightly modified definition fixes the problem:

```
\documentclass{article}

\usepackage{xcookybooky}

\renewcommand{\step}
{%
    \stepcounter{step}
    \lettrine
    [%
        lines=2,
        lhang=0,          % space into margin, value between 0 and 1
        loversize=0.15,   % enlarges the height of the capital
        slope=0em,
        findent=1em,      % gap between capital and intended text
        nindent=0em       % shifts all intended lines, begining with the second line
    ]{\thestep}{}%
}


\begin{document}

\begin{recipe}{Back}  
  \preparation{%
		\step test
  }
\end{recipe}

\end{document} 
```

**Comment by samcarter** [comment]
This issue can be closed -- it was fixed in https://github.com/SvenHarder/xcookybooky/pull/14

**Comment by ClaudiusL** [comment]
Yes, the issue has been fixed in 2018 -- but the version on CTAN still dates back to 2015. Is there a new CTAN release planned any time soon?

**Comment by 52buickman** [comment]
Must be broke again. . .  I'm running version 2.23.  Strangely I can put the lettrine code in my script and it works.  I don't see why the xcookybooky code doesn't work.  Possibly and issue with \thestep?  This works:

       \lettrine[lines=2,lhang=0,loversize=0.15,slope=0em,findent=1em,nindent=0em]{b}{lah blah blah}

**Comment by samcarter** [comment]
@52buickman  It does not matter that you run lettrine 2.23, what matters is the version of xcookybooky. You will need the development version of xcookybooky here from github. While the necessary changes were made here, they never made it to ctan

**Comment by 52buickman** [comment]
@samcarter Ah! Got it.  I assumed that CTAN was up to date.  The change makes sense.  Thanks.  Though I'm new to LaTeX development, I've programmed over much of the the past 40 years.  Help me to understand.  What is involved to get the current version promoted into CTAN?

**Comment by samcarter** [comment]
@52buickman  Basically getting CTAN up to date requires the package author to sent them a new version (with a couple of exceptions in case the author goes MIA)

**Comment by kaugummiheld** [comment]
> You will need the development version of xcookybooky here from github
Hey, I stumbled over this exact problem when giving xcookybooky a try.
Could somebody explain how to "update" to this version here on github? I'm using xcookybooky via `texlive-latexextra` I guess. Thanks a lot!

**Comment by samcarter** [comment]
@kaugummiheld  Go to https://github.com/SvenHarder/xcookybooky and click on the green "Code" button to download the code. Then go into this folder and run `tex xcookybooky.ins`, this will create the `xcookybooky.sty` file. Place this file in the same folder as your .tex file.

**Comment by kaugummiheld** [comment]
@samcarter Thanks for explaining. I wasn't sure on what location to put the files.
Maybe a short note somewhere in the Readme would be helpful?

**Comment by samcarter** [comment]
> Maybe a short note somewhere in the Readme would be helpful?

Not really. This is just a quick workaround until a new version is released. The proper way is clearly the maintainer uploading to ctan and not individual users trying to install packages manually. That's very much against the philosophy of latex
